src/util/control_buttons.js

```diff
@@ -0,0 +1,63 @@
+(function () {
+  /**
+   * Fetch the SVG belonging to a specific RemixIcon.
+   *
+   * @param {string} cssClass - The RemixIcon class of the requested icon
+   * @returns {string|undefined} The SVG path of the associated RemixIcon if it exists
+   */
+  const getIconPath = async function (cssClass) {
+    const url = browser.runtime.getURL('/src/lib/remixicon-svg.json');
```

Comment:
missed a renamed path

```suggestion
    const url = browser.runtime.getURL('/src/lib/remixicon_svg.json');
```